### DISCUSSION TITLE
Need Help with Disabling CSS chunk

### DISCUSSION THREAD

**Discussion by Croodo** [nuxt/nuxt#19318] [Questions]
by default, CSS is chunked in different files but I want to use CSS directly in HTML inside `<style></style>` .

In the documentation, webpack config is available but how to do it in vite?

```
export default {
  webpack: {
    extractCSS: true,
    // or
    extractCSS: {
      ignoreOrder: true
    }
  }
}
```

**Comment by Baroshem** [comment]
Hey,

Check the following configuration from vite that you should be able to pass to the nuxt config (vite object) -> https://vitejs.dev/guide/features.html#css-code-splitting

```ts
{
  vite: {
   build: {
     cssCodeSplit: false
   }
  }
}
```

**Comment by Croodo** [comment]
Don't know why it is not working .
Do I need setup vite.config  file? 
nuxt.config 
```
export default defineNuxtConfig({
  vite: {
   build: {
     cssCodeSplit: false
   }
  },
  css: ['~/assets/css/main.css'],

  postcss: { 
    plugins: {
      tailwindcss: {},
      autoprefixer: {},

    },
  }, .....
```

**Reply by Baroshem** [reply]
You should be able to pass the configuration to vite from nuxt config. 

This code you mentioned probably wont work as Tailwind by default uses the purgeCSS unles you intruct it to not use it in certain files and will remove css that is not necessary.

You should probably take a look at this as well https://tailwindcss.com/docs/content-configuration
